dashboard/app/controllers/census_reviewers_controller.rb

```diff
@@ -0,0 +1,54 @@
+class CensusReviewersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :require_census_reviewer_permissions
+
+  def require_census_reviewer_permissions
+    return if current_user.permission?(UserPermission::CENSUS_REVIEWER)
+    raise(CanCan::AccessDenied, 'user must be a census reviewer')
+  end
```

Comment:
This should be a CanCan ability, defined in `ability.rb`, and checked via `authorize_resource` rather than these 2 custom `before_action` calls.

Reply:
Done.

Reply:
@aoby can you take a look at [the change I made to use a CanCan ability](https://github.com/code-dot-org/code-dot-org/pull/21379/commits/e46ed2fc35368b29838aa6c4f1284315358c52d6) and let me know if it looks okay? I wasn't familiar with this and want to make sure I got it right. It works properly, but I'm not sure if I missed any conventions.